bin/api-docs/gen-block-lib-list.js

```diff
@@ -155,14 +155,20 @@ function readBlockJSON( filename ) {
 			? processObjWithInnerKeys( augmentSupports( blockjson.supports ) )
 			: [];
 	const attributes = getTruthyKeys( blockjson.attributes );
+	const parent = blockjson.parent
+		? '\n' + `-	**Parent:** ${ blockjson.parent }`
```

Comment:
```suggestion
		? '\n' + `-	**Parent:** ${ blockjson.parent.join( ', ' ) }`
```

This code does not cause anything for now, but in the future, if the `parent` property has more than one parent, it will add a single space after the comma.